src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.netstd.cs

```diff
@@ -112,6 +112,30 @@ protected override Foundation.Size ArrangeOverride(Foundation.Size finalSize)
 			return finalSize;
 		}
 
+		protected override void OnLoaded()
+		{
+			base.OnLoaded();
+			RegisterEventHandler("scroll", (EventHandler)OnScroll);
+		}
+
+		protected override void OnUnloaded()
+		{
+			base.OnUnloaded();
+			UnregisterEventHandler("scroll", (EventHandler)OnScroll);
+		}
+
+		private void OnScroll(object sender, EventArgs args)
+		{
+			int.TryParse(GetProperty("scrollLeft"), out var horizontalOffset);
+			int.TryParse(GetProperty("scrollTop"), out var verticalOffset);
```

Comment:
You can extract those information in the `eventExtractorScript` to prevent interop.

Reply:
Nice, I'll try that.